crowdsourcing/viewsets/user.py

```diff
@@ -155,3 +158,33 @@ def retrieve(self, request, *args, **kwargs):
         user = get_object_or_404(self.queryset, user=request.user)
         serializer = UserPreferencesSerializer(instance=user)
         return Response(serializer.data)
+
+
+class CountryViewSet(viewsets.ModelViewSet):
```

Comment:
when using a ModelViewset it will inherit 5 default actions -> endpoints 
.list(), .retrieve(), .create(), .update(), and .destroy() that means that you wont need the `get_country` in this particular case, and since you have a serializer_class you can use `Response(data=..., status=....)`

Best is to use mixins and only inherit the actions you need  see `UserViewSet`